saltgui/static/scripts/output.js

```diff
@@ -21,6 +21,63 @@
 
 class Output {
 
+  // format a JSON object
+  // based on an initial indentation and an indentation increment
+  static _formatJSON0(value, indentStr, indentCnt) {
```

Comment:
Can we just have a `static formatJSON(json, indent=4)` function here instead of a public and private one?

Difference between initial indentation and other indentation would be something I avoid. The indentation param should be a number and not a string.

Reply:
Of course. I'm not used to languages with actual default values for their parameters. Hence the strict separation.

Reply:
I kept the the indentation parameters as string. the are used in multiple places and I don't want to use `' '.repeat(indentCnt)` each time.